### DISCUSSION TITLE
No access to network drives

### DISCUSSION THREAD

**Discussion by VamLykTan** [brunoherbelin/vimix#132] [Q&A]
Hello @ll

I was just about to set up ViMix on Ubuntu 22 and load all the videos and images for my next stream on Twitch and get a little familiar with the environment, but I noticed that my 2 network drives were not listed. See Appendix 1
![Bildschirmfoto vom 2024-03-14 09-51-00](https://github.com/brunoherbelin/vimix/assets/6794318/09304bab-139d-4738-b20e-6c8c11591915)

However, a look at Nautilus tells me that all 4 SDB drives are loaded and ready. See Appendix 2
![Bildschirmfoto vom 2024-03-14 10-06-56](https://github.com/brunoherbelin/vimix/assets/6794318/a78a9626-db92-440a-a3c9-5358666ec134)

Is this intentional, or am I just missing a setting in VImix?

I'm asking because as a DJ/VJ I have to keep my video material for up to 3 computers in the network and I DON'T want to have it individually on each computer.

**Comment by brunoherbelin** [comment]
Well, nothing intentionally done to prevent anyone to access any drive :) 
Two things which could help:

-  did you check the [application permissions](https://github.com/brunoherbelin/vimix/wiki/Quick-Installation-Guide#set-app-permissions) for vimix flapak ?
-  you can also drag&drop files from nautilus directly in vimix workspace

**Comment by VamLykTan** [comment]
2 Second way will probably be the most effective way.

Because of access rights, I'll take another look at it this weekend.

But thank you very much for the program. If desired, I would be happy to offer you support for a German translation.
However, I have to say that I only work with FreePascal and therefore have no CPP environment installed
